splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/ScanOperation.java

```diff
@@ -34,12 +34,13 @@
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import org.apache.log4j.Logger;
 
+import java.io.Externalizable;
 import java.io.IOException;
 import java.io.ObjectInput;
 import java.io.ObjectOutput;
 import java.util.Arrays;
 
-public abstract class ScanOperation extends SpliceBaseOperation{
+public abstract class ScanOperation extends SpliceBaseOperation implements Externalizable {
```

Comment:
Why do we need this one to be externalizable?

Reply:
Removed